src/openforms/registrations/contrib/objects_api/tests/test_templatetags.py

```diff
@@ -0,0 +1,68 @@
+from unittest import TestCase
+
+from django.template import Context, Template
+
+from openforms.registrations.constants import RegistrationAttribute
+from openforms.submissions.tests.factories import SubmissionFactory
+
+
+class JsonSummeryTests(TestCase):
+    def test_render_json_summary(self):
+        submission = SubmissionFactory.from_components(
+            [
+                {
+                    "key": "voornaam",
+                    "type": "textfield",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_voornamen,
+                    },
+                },
+                {
+                    "key": "achternaam",
+                    "type": "textfield",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_geslachtsnaam,
+                    },
+                },
+                {
+                    "key": "tussenvoegsel",
+                    "type": "textfield",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_tussenvoegsel,
+                    },
+                },
+            ],
+            submitted_data={
+                "voornaam": "Foo",
+                "achternaam": "Bar",
+                "tussenvoegsel": "de",
+            },
+            language_code="en",
+        )
+        submission_step = submission.steps[0]
+        step_slug = submission_step.form_step.form_definition.slug
+
+        # Loads template tag and uses template tag
+        template = Template(
+            """{% load registrations.contrib.objects_api.json_summary %}{% json_summary %}"""
+        )
+
+        rendered = template.render(Context({"_submission": submission}))
+
+        expected = (
+            '{"%s": {"voornaam": "Foo", "achternaam": "Bar", "tussenvoegsel": "de"}}'
+            % step_slug
+        )
+
+        self.assertEqual(rendered, expected)
+
+    def test_render_json_summary_no_submission(self):
+        # Loads template tag and uses template tag
+        template = Template(
+            """{% load registrations.contrib.objects_api.json_summary %}{% json_summary %}"""
+        )
+
+        rendered = template.render(Context())
+        expected = "{}"
+
+        self.assertEqual(rendered, expected)
```

Comment:
```suggestion
        rendered = render_from_string("{% json_summary %}")

        self.assertEqual(rendered, "{}")
```

The loading of template tag libraries is not something that we want to burden our form designers with, that's why they are implicitly loaded in the template engine by us for them.

Next you want to make sure that these templates can be loaded with the template backend that will actually be used, and you can still use that to test the behaviour.

With simple assertions, please just use the literals directly which makes the test shorter and easier to follow.